_components/icon/guidance/usability.md

```diff
@@ -1,3 +1,8 @@
-- **Combine with text.** Users will not always understand the meaning of icons alone. Use icons combined with text to improve usability.
-- **Be consistent with icon meaning.** Icons that are used a number of times throughout an application or site must have a consistent meaning and text description throughout. For example, if an icon of a blank piece of paper means “new document” on most screens, the same icon shouldn’t be used elsewhere to mean “reformat document.” Consistency aids users with cognitive disabilities and creates a better user experience for all users.
-- **When interactive, combine with other components.** If the icon is part of an interactive element, it should be implemented within another functional component, like as part of a button or list.
\ No newline at end of file
+{% capture home_icon %}<svg class="usa-icon bottom-neg-2px" aria-hidden="true" focusable="false" role="img"><use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#home"></use></svg>{% endcapture %}
+{% capture print_icon %}<svg class="usa-icon bottom-neg-2px" aria-hidden="true" focusable="false" role="img"><use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#print"></use></svg>{% endcapture %}
+{% capture search_icon %}<svg class="usa-icon bottom-neg-2px" aria-hidden="true" focusable="false" role="img"><use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#search"></use></svg>{% endcapture %}
+
+{:.usa-content-list }
+- **Combine icons with text.** Only a few icons are consistently understood across the digital-using public of the world, among them home ({{ home_icon }}), print ({{ print_icon }}), and search ({{ search_icon }}). Combine icons with text to improve clarity, and test your icons for recognition and memorability with your particular audience.
+- **Be consistent with icon meaning.** Icons used more than once in an application or site must be used to represent the same thing, and have the same text description in every instance. For example, if an icon of a blank piece of paper means “new document” on most screens, choose a different icon to communicate “reformat document.” Consistency helps people with some cognitive disabilities, people who might be distracted or scanning, and creates a better user experience for all.
```

Comment:
I recommend adding the word "helps" to give this list a more parallel structure: Consistency helps people with some cognitive disabilities, **helps** people who might be distracted or scanning, and creates a better user experience for all.

Reply:
Good note. Updated in [da7f69a](https://github.com/uswds/uswds-site/pull/2643/commits/da7f69aac35b791daec3f23124e5667af1038f94). Let me know if you need anything else!

Reply:
Looks good--thanks, @amyleadem!